pulp_container/app/registry_api.py

```diff
@@ -309,13 +312,21 @@ def get_pull_through_drv(self, path):
         if not pull_through_cache_distribution:
             raise RepositoryNotFound(name=path)
 
+        pull_through_remote = models.ContainerPullThroughRemote.objects.filter(
+            pk=pull_through_cache_distribution.remote_id
+        ).values()[0]
```

Comment:
I think we can access `pull_through_remote` directly from `pull_through_cache_distribution.remote`. To save one DB query, we can run this a couple of lines above:

        pull_through_cache_distribution = (
            models.ContainerPullThroughDistribution.objects.annotate(path=Value(path))
            .filter(path__startswith=F("base_path"))
            .order_by("-base_path")
            .select_related("remote")
            .first()
        )